### PR TITLE
PLATML 4813 - changed the name of output dataset id

### PR DESCRIPTION
**Pull request by grujicaleksandar**
Changed the name of output dataset id from **output_dataset_id** to **scoringResultsDataSetId**

### CODE REVIEW

**Review by rjrussell77 — APPROVED** [review]
LGTM

**Comment by rjrussell77** [comment]
We won't need all the reviewers on the list to give the OK here.  Whoever has the most exposure to this code, please give the ok and let's move this along.

File: recipes/R/Retail - GradientBoosting/retail.config.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
         "name": "score",
         "parameters": [
             {
-                "key": "output_dataset_id",
+                "key": "scoringResultsDataSetId",
```
Comment (shivmpatel12) [inline recipes/R/Retail - GradientBoosting/retail.config.json]: This config can be removed entirely. I'm also referring to your question in the Jira ticket. `scoringResultsDataSetId` is not picked by the UI, by it is set by the UI. When you choose a scoring results dataset, the UI passes in the id for that dataset as `scoringResultsDataSetId` in the configs. And as Matt mentioned in the Jira, which scoring results datasets you see in the UI depend on `ACP_DSW_SCORING_RESULTS_XDM_SCHEMA`. But due to some recent UI improvements, this config is also set by the UI when you are creating a recipe and choose an output schema, so you should actually be able to remove `ACP_DSW_SCORING_RESULTS_XDM_SCHEMA` too.

Comment (grujicaleksandar) [reply recipes/R/Retail - GradientBoosting/retail.config.json]: Thank you for explaining this.

**Review by shivmpatel12 — APPROVED** [review]
Looks great!

**Review by tumulurik — APPROVED** [review]
LGTM!

**Pull request merged**